test/src/test_listMatches.cpp

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2019 The Nakama Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "test_main.h"
+
+namespace Nakama {
+namespace Test {
+
+using namespace std;
+
+class NMatchListTest : public NCppTest
+{
+    NSessionPtr session;
+
+public:
+    explicit NMatchListTest(const char* name) : NCppTest(name) {}
```

Comment:
is the `explicit` keyword necessary?

Reply:
Followed the same approach as one of the Groups tests but happy to remove if it's not needed. https://github.com/heroiclabs/nakama-cpp/blob/master/test/src/test_groups.cpp#L123